src/applications/disability-benefits/all-claims/pages/medals.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../content/ptsdClassification';
+import { MedalsDescription } from '../content/medals';
+
+export const uiSchema = index => ({
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781" />
+  ),
+  'ui:description': ({ formData }) => (
+    <MedalsDescription formData={formData} index={index} formType="781" />
+  ),
+  [`medals${index}`]: {
+    'ui:title': ' ',
+    'ui:widget': 'yesNo',
+  },
+  [`medalType${index}`]: {
+    'ui:title': 'Please tell us what medal or citation you received. ',
+    'ui:options': {
+      expandUnder: `medals${index}`,
+      expandUnderCondition: true,
+    },
+  },
+});
+
+export const schema = index => ({
+  type: 'object',
+  properties: {
+    [`medals${index}`]: {
+      type: 'boolean',
+      properties: {},
+    },
+    [`medalType${index}`]: {
+      type: 'string',
```

Comment:
Is there supposed to be any type of character limit on this text box?

Reply:
There's currently no max lengths set on this per schema.